src/test/java/org/folio/linked/data/test/TestUtil.java

```diff
@@ -125,4 +130,28 @@ public static void awaitAndAssert(ThrowingRunnable throwingRunnable) {
       .pollInterval(ONE_HUNDRED_MILLISECONDS)
       .untilAsserted(throwingRunnable);
   }
+
+  public static void assertAuthority(Resource resource,
+                               String label,
+                               boolean isActive,
+                               boolean isPreferred,
+                               Resource replacedBy) {
+    assertThat(resource)
+      .hasFieldOrPropertyWithValue("label", label)
+      .hasFieldOrPropertyWithValue("active", isActive)
+      .satisfies(r -> assertThat(r.getDoc()).isNotEmpty())
```

Comment:
minor
looks like redundant

Reply:
but if we omit it, next line will throw NPE in case r.getDoc() is null, right? So let's keep it